play-2.4/swagger-play2/app/play/modules/swagger/SwaggerPlugin.scala

```diff
@@ -80,6 +80,24 @@ class SwaggerPluginImpl @Inject()(lifecycle: ApplicationLifecycle, router: Route
     case Some(value)=> value
   }
 
+  // Play seems to throw an exception if this value is not defined and you get it
```

Comment:
Actually as far as I can tell the Play Configuration class is well and truly broken. It fails to properly convert the values to Option. The Typesafe Config class allows you to handle all the following 3 cases correctly: the key was not defined, the key was defined but was set to null and the key was defined. Play should convert this to None, Some(null) and Some(value) but currently Play is only capable of returning Some(value).